[2023-06-15T22:52:03.215+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [queued]>
[2023-06-15T22:52:03.221+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [queued]>
[2023-06-15T22:52:03.221+0200] {taskinstance.py:1308} INFO - Starting attempt 8 of 8
[2023-06-15T22:52:03.238+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_participants> on 2023-06-15 14:47:07.980597+00:00
[2023-06-15T22:52:03.242+0200] {standard_task_runner.py:57} INFO - Started process 116551 to run task
[2023-06-15T22:52:03.245+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ejecutar_python', 'check_participants', 'manual__2023-06-15T14:47:07.980597+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/thesis_container.py', '--cfg-path', '/tmp/tmpzsl5591p']
[2023-06-15T22:52:03.247+0200] {standard_task_runner.py:85} INFO - Job 52: Subtask check_participants
[2023-06-15T22:52:03.283+0200] {task_command.py:410} INFO - Running <TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:47:07.980597+00:00 [running]> on host DESKTOP-3KJU1M7.
[2023-06-15T22:52:03.345+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ejecutar_python' AIRFLOW_CTX_TASK_ID='check_participants' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T14:47:07.980597+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T14:47:07.980597+00:00'
[2023-06-15T22:52:03.346+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - ['']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - New participants found. Creating containers...
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - 0
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - <class 'int'>
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - prueba de lista ['hola_525']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/host_2/1_6-6.mrxs']
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - container solito hola_525
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-15T22:52:03.347+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2
[2023-06-15T22:52:03.348+0200] {logging_mixin.py:149} INFO - nuevo /home/reynel1995/Thesis/hola_525
[2023-06-15T22:52:03.348+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/hola_525', '', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525', '/home/reynel1995/Thesis/hola_525']
[2023-06-15T22:52:03.875+0200] {logging_mixin.py:149} INFO - Container hola_525 image built successfully.
[2023-06-15T22:52:03.976+0200] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 319, in check_participants
    build_and_run_containers(index)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 147, in build_and_run_containers
    subprocess.run(run_command, shell=True, check=True)
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'docker run -d --name hola_525 hola_525' returned non-zero exit status 125.
[2023-06-15T22:52:03.985+0200] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=ejecutar_python, task_id=check_participants, execution_date=20230615T144707, start_date=20230615T205203, end_date=20230615T205203
[2023-06-15T22:52:04.005+0200] {standard_task_runner.py:104} ERROR - Failed to execute job 52 for task check_participants (Command 'docker run -d --name hola_525 hola_525' returned non-zero exit status 125.; 116551)
[2023-06-15T22:52:04.020+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-15T22:52:04.050+0200] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
